packages/tools/pluggable-widgets-tools/src/typings-generator/generateClientTypes.ts

```diff
@@ -46,20 +46,30 @@ function generateClientTypeBody(
     generatedTypes: string[],
     resolveProp: (key: string) => Property | undefined
 ) {
+    function actionIsLinkedInAnAttribute(key: string): boolean {
+        return properties.some(prop => prop.$.type === "attribute" && prop.$.onChange === key);
```

Comment:
This won't work if the property path is relative, for example if the attribute property is in an object list and its `onChange` action is defined in the surrounding properties. Then the `onChange` attribute in the XML will have at least one `../` prefix.

Reply:
Fixed now :)